### PR TITLE
Added Sethome Command

### PR DESCRIPTION
**Pull request by MertUnverdi**

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/configs/Messages.java [hunk 1]
```diff
@@ -18,5 +18,7 @@
     public String islandSchematicNotFound = "%prefix% &7No schematic with that name exists.";
     public String islandDeleted = "%prefix% &7Your island has been deleted.";
     public String teleportingHome = "%prefix% &7Teleporting Home...";
+    public String setHome = "%prefix% &7You have set your home to this location.";
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/configs/Messages.java:21]: Change to Your Island  home has been set

File: src/main/java/com/iridium/iridiumskyblock/commands/SetHomeCommand.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+     * @param sender The CommandSender which executes this command
+     * @param args   The arguments used with this command. They contain the sub-command
+     */
+    @Override
+    public void execute(CommandSender sender, String[] args) {
+        Player player = (Player) sender;
+        User user = IridiumSkyblockAPI.getInstance().getUser(player);
+        if (user.getIsland() != null) {
+            // TODO: There should be a is safe method
+            if (true) {
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/commands/SetHomeCommand.java:39]: create a method in  IslandManager called isLocationSafe that takes  in an  island and a location
(just return true for now, ill implement it tomorrow)

**Review by PeachesMLG — CHANGES_REQUESTED** [review]

File: src/main/java/com/iridium/iridiumskyblock/configs/Messages.java (hunk 1 above)
Comment (MertUnverdi) [reply src/main/java/com/iridium/iridiumskyblock/configs/Messages.java:21]: oh, i was copied it from old messages -,-

File: src/main/java/com/iridium/iridiumskyblock/commands/SetHomeCommand.java (hunk 2 above)
Comment (MertUnverdi) [reply src/main/java/com/iridium/iridiumskyblock/commands/SetHomeCommand.java:39]: okay i'll do

**Review by PeachesMLG — APPROVED** [review]

**Pull request merged**